eng/WpfArcadeSdk/tools/CodeAnalysis/WpfCodeAnalysis.ruleset

```diff
@@ -19,4 +19,7 @@
   <Rules AnalyzerId="System.Runtime.CSharp.Analyzers" RuleNamespace="System.Runtime.CSharp.Analyzers">
     <Rule Id="CA2213" Action="None" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.FxCopAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.FxCopAnalyzers">
+    <Rule Id="CA1062" Action="None" />
```

Comment:
Disable [CA1062](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1062) because there are false positives, when a public method is invoking another method that already does the null check.

Reply:
@ThomasGoulet73  There are only two such cases where this warning is raised. Shall we suppress only for those two locations, instead of globally suppressing it?

Reply:
@dipeshmsft You're right, I thought about it more and I think we should enable it to make sure that we do null checks in the public method, even if there is a null check in a private method. This helps make sure that the argument name in the exception is the same as the public one, in case the argument name in the private method is not the same as the one in the public method. This additional check probably won't even be noticeable performance-wise.

I'll enable it in a separate commit and I'll leave the decision to the reviewers.